src/twisted/conch/ssh/transport.py

```diff
@@ -94,17 +94,13 @@ class SSHCiphers:
 
     cipherMap = {
         b"3des-cbc": (algorithms.TripleDES, 24, modes.CBC),
-        b"blowfish-cbc": (algorithms.Blowfish, 16, modes.CBC),
```

Comment:
Just a question.

Can we have them still supported, but removed by default?

Reply:
Yes. Deprecated ciphers are in another list/dict while the original cipher list is kept.